spyderlib/requirements.py

```diff
@@ -39,7 +39,7 @@ def check_qt():
     try:
         import qtpy
         package_name, required_ver = qt_infos[qtpy.API]
-        actual_ver = qtpy.__version__
+        actual_ver = qtpy.PYQT_VERSION

```

Comment:
I know we don't support pyside officially anymore, but if I'm not mistaken this will break pyside, since `qtpy.PYQT_VERSION` is None with pyside. I think using `qtpy.QT_VERSION` instead should do the trick.